app/javascript/dashboard/components-next/Contacts/ContactsSidebar/components/ContactNoteItem.vue

```diff
@@ -56,8 +82,28 @@ const handleDelete = () => {
       />
     </div>
     <p
+      ref="noteContentRef"
       v-dompurify-html="formatMessage(note.content || '')"
-      class="mb-0 prose-sm prose-p:mb-1 prose-p:mt-0 prose-ul:mb-1 prose-ul:mt-0 text-n-slate-12"
+      class="mb-0 prose-sm prose-p:text-sm prose-p:leading-relaxed prose-p:mb-1 prose-p:mt-0 prose-ul:mb-1 prose-ul:mt-0 text-n-slate-12"
+      :class="{
+        'line-clamp-4': collapsible && !isExpanded && needsCollapse,
+      }"
     />
+    <p v-if="collapsible && needsCollapse">
```

Comment:
@scmmishra In some cases, even when the collapsible button isn't needed, it still appears. You can see it in the video below. Could you please take a look?

https://github.com/user-attachments/assets/ff334824-ac03-49f7-8259-5e6401ce1659